payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json

```
@@ -0,0 +1,1728 @@
+								"exec": [
+									""
+								]
+							}
+						}
+					]
+				},
+				{
+					"name": "3. Create transfer",
+					"item": [
+						{
+							"name": "1. Update quote with selected recipient",
+							"request": {
+								"method": "PATCH",
+								"header": [
+									{
+										"key": "Authorization",
+										"value": "Bearer {{token}}"
+									},
+									{
+										"key": "Content-Type",
+										"name": "Content-Type",
+										"type": "text",
+										"value": "application/merge-patch+json"
+									}
+								],
+								"body": {
+									"mode": "raw",
+									"raw": "{\n    \"targetAccount\": {{new-recipient-id}}\n}"
+								},
+								"url": {
+									"raw": "{{host}}/v2/quotes/{{new-quote-id}}",
+									"host": [
+										"{{host}}"
+									],
+									"path": [
+										"v2",
+										"quotes",
+										"{{new-quote-id}}"
+									]
+								},
+								"description": "Set the recipient the user wishes to send to on the quote to get an updated price and delivery estimte.\n\nENSURE `Content-Type` IS `application/merge-patch+json` ELSE THIS WILL FAIL"
+							},
+							"response": []
+						},
+						{
+							"name": "2. Get transfer extra info dynamic form",
+							"request": {
+								"method": "POST",
+								"header": [
+									{
+										"key": "Authorization",
+										"value": "Bearer {{token}}"
+									},
+									{
+										"key": "Content-Type",
+										"value": "application/json",
+										"type": "text"
+									}
+								],
+								"body": {
+									"mode": "raw",
+									"raw": "{\n    \"targetAccount\": {{new-recipient-id}},\n    \"quoteUuid\": \"{{new-quote-id}}\"\n}"
+								},
+								"url": {
+									"raw": "{{host}}/v1/transfer-requirements",
+									"host": [
+										"{{host}}"
+									],
+									"path": [
+										"v1",
+										"transfer-requirements"
+									]
+								}
+							},
+							"response": []
+						},
+						{
+							"name": "3. Update transfer extra info dynamic form (if any fields require refresh)",
+							"request": {
+								"method": "POST",
+								"header": [
+									{
+										"key": "Authorization",
+										"value": "Bearer {{token}}"
+									},
+									{
+										"key": "Content-Type",
+										"value": "application/json"
+									}
+								],
+								"body": {
+									"mode": "raw",
+									"raw": "{ \n\t\"targetAccount\": {{new-recipient-id}},\n    \"quoteUuid\": \"{{new-quote-id}}\",\n    \"details\": {\n        \"reference\": \"my ref\",\n    \t\"sourceOfFunds\": \"verification.source.of.funds.other\"\n    }\n}"
+								},
+								"url": {
+									"raw": "{{host}}/v1/transfer-requirements",
+									"host": [
+										"{{host}}"
+									],
+									"path": [
+										"v1",
+										"transfer-requirements"
+									]
+								}
+							},
+							"response": []
+						},
+						{
+							"name": "4. Generate GUID for idempotency",
```

```suggestion
							"name": "4. Generate GUID for idempotency - payouts only",
```

This isn't required for 3PP, not sure if we can word it better though...